templates/web-deployment.yaml

```diff
@@ -603,13 +603,17 @@ spec:
             - name: CONCOURSE_VAULT_CA_CERT
               value: "{{ .Values.web.vaultSecretsPath }}/ca.cert"
             {{- end }}
-            {{- if eq .Values.concourse.web.vault.authBackend "token" }}
+            {{- if eq .Values.concourse.web.vault.authBackend "token" and eq .Values.concourse.web.vault.tokenPath ""}}
             - name: CONCOURSE_VAULT_CLIENT_TOKEN
               valueFrom:
                 secretKeyRef:
                   name: {{ template "concourse.web.fullname" . }}
                   key: vault-client-token
             {{- end }}
+            {{- if not eq .Values.concourse.web.vault.tokenPath ""}}
```

Comment:
Hi @jenniferplusplus , after i merged it I realized I should probably find an example of using `if not eq xxx ""` to check a non-empty string. Refer to this [answer](https://stackoverflow.com/questions/67706931/helm-template-check-for-empty-string), i am not sure current one would work without grouping it like `if not (eq xxx "")`.

Given the way we check for existing path value (line 635), I think just do `if xxx` for this line and `if not xxx` for line 606 might be better. If you think so, would might open another PR for it? Thank you!

Reply:
Sure, I can do that. I'm really not familiar with building helm charts, so I was trying to just follow the existing patterns.

Reply:
@xtremerui fyi, in case you didn't see it, I opened #358 for this issue